scripts/check_tests.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
The acceptance test steps already fail if the tests fail, and that failure is reflected in both the summary view of the GitHub Actions workflow (there's a red circle with an x next to the step if it fails) and in the console output for the test step.  Rather than add this missing check script, why not remove the references to it instead?